gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/inspection/inspectResponses.ts

```diff
@@ -2,9 +2,39 @@ import { Result } from '@chainlink/gauntlet-core'
 import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
 import { PublicKey } from '@solana/web3.js'
 import { CONTRACT_LIST, getContract } from '../../../../lib/contracts'
-import { inspection } from '@chainlink/gauntlet-core/dist/utils'
+import { inspection, logger } from '@chainlink/gauntlet-core/dist/utils'
+import { getLatestNewTransmissionEvents } from '../../../../lib/events'
+import { toComparablePubKey } from '../../../../lib/inspection'
+import RDD from '../../../../lib/rdd'
 
-type Input = {}
+type Input = {
+  description: string
+  aggregatorOracles: Oracle[]
+}
+
+type Oracle = {
+  transmitter: string
+  website: string
+  name: string
+  apis: string[]
+  observation?: number
+}
+
+// Returns a formatted oracle log given an address, a command input, a label, and a start tab,
+const oracleLog = (address: string, label: string, startTab: number, input?: Input) => {
```

Comment:
We can simplify and make a bit clearer the interface of this. Receiving the oracle to print or the list of oracles seems cleaner. We can give a default value to the `startTab` as well.
```suggestion
const makeOracleLog = (address: string, label: string, oracles: Oracle[], startTab = 2) => {
```